README.md

```diff
@@ -57,13 +57,12 @@ The following table lists equivalent API function signatures in v2.0.0 (Release
 | v2.0.0 (Release Candidate) | v3.0.0 | Notes |
 | :-: | :-: | :-: |
 | `OtaState_t OTA_Shutdown( uint32_t ticksToWait );` | `OtaState_t OTA_Shutdown( uint32_t ticksToWait, uint8_t unsubscribeFlag );` | `unsubscribeFlag` indicates if unsubscribe operations should be performed from the job topics when shutdown is called. Set this as 1 to unsubscribe, 0 otherwise. |
-## Reference examples
 
-Please refer to the demos of the AWS IoT Over-the-air Updates library in the following location for a reference example on POSIX:
+### How to migrate from version 1.0.0 to version 3.0.0 for OTA applications
+
+Refer to [OTA Migration document](https://docs.aws.amazon.com/freertos/latest/portingguide/porting-migration-ota.html) for the summary of updates to the API.
+[Migration document for OTA PAL](https://docs.aws.amazon.com/freertos/latest/portingguide/porting-migration-ota-pal.html) also provides a summary of updates required for upgrading the OTA-PAL to work with v3.0.0 of the library.
 
-| Platform | Location |
-| :-: | :-: |
-| POSIX | [AWS IoT Device SDK for Embedded C](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/main/demos/ota) |
 ## Porting
 
 In order to support AWS IoT Over-the-air Updates on your device, it is necessary to provide the following components:
```

Comment:
MQTT is required but HTTP is optional and only required when HTTP is used for file download

Reply:
Good catch, updated.

Reply:
Thanks , but can we clarify that when MQTT is used for download then only MQTT interface is required , but for HTTP download both MQTT and HTTP is required as MQTT in this case is used for job notifications and status updates whereas HTTP is for file download

Reply:
Done